src/styles/blocks/card-but.scss

```diff
@@ -0,0 +1,19 @@
+.card__buy {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.price {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md). Change this in all places in your code.